js/client-battle-tooltips.js

```diff
@@ -343,7 +348,20 @@ var BattleTooltips = (function () {
 				text += '&nbsp;SpA /&nbsp;' + myPokemon.stats['spd'] + '&nbsp;SpD /&nbsp;';
 			}
 			text += myPokemon.stats['spe'] + '&nbsp;Spe</p>';
-			if (isActive) text += '<p class="section">Opponent sees:</p>';
+			if (isActive) {
+				pokemon.stats = myPokemon.stats;
+				pokemon.ability = ability;
+				pokemon.item = item;
+				pokemon.hp = myPokemon.hp;
+				pokemon.maxhp = myPokemon.maxhp;
+				var modifiedStats = this.calculateModifiedStats(pokemon);
```

Comment:
Once again, pass both pokemon, and myPokemon; the reason they're separate is because we _don't_ want the data to be merged. One is publicly-known battle state and one is privately known team data.